src/links.ts

```diff
@@ -8,14 +8,18 @@ export function convertInternalLinks(
   layoutStrategy: LayoutStrategy
 ): string {
   const convertHref = (url: string) => {
-    const p = pages.find(p => {
+    const page = pages.find(p => {
       return p.matchesLinkId(url);
     });
-    if (p) {
-      verbose(
-        `Converting Link ${url} --> ${layoutStrategy.getLinkPathForPage(p)}`
-      );
-      return layoutStrategy.getLinkPathForPage(p);
+    if (page) {
+      let convertedLink = layoutStrategy.getLinkPathForPage(page);
+
+      // Include the fragment (# and after) if it exists
+      const { fragmentId } = parseLinkId(url);

```

Comment:
I guess you don't need `baseLinkId` here too, just to keep the destructuring happy?